tile.yml

```diff
@@ -63,17 +63,18 @@ packages:
     env:
       GOPACKAGENAME: github.com/wavefronthq/cloud-foundry-nozzle-go
 
-      NOZZLE_API_URL: (( .properties.pcf_host.value ))
-      NOZZLE_USERNAME: (( .properties.pcf_user.value ))
-      NOZZLE_PASSWORD: (( .properties.pcf_password.value ))
-      NOZZLE_LOG_STREAM_URL: (( .properties.pcf_log_stream.value ))
+      NOZZLE_API_URL: https://api.(( ..cf.cloud_controller.system_domain.value ))
+      NOZZLE_USERNAME: admin
+      NOZZLE_PASSWORD: (( ..cf.uaa.admin_credentials.password ))
+      NOZZLE_LOG_STREAM_URL: https://log-stream.(( ..cf.cloud_controller.system_domain.value ))
 
-      NOZZLE_FIREHOSE_SUBSCRIPTION_ID: firehose-subscription-id
-      NOZZLE_SKIP_SSL: (( .properties.pcf_skip_ssl_validation.value ))
+      NOZZLE_FIREHOSE_SUBSCRIPTION_ID: wf-firehose-subscription-id
+      NOZZLE_SKIP_SSL: true
 
       WAVEFRONT_PROXY_PORT: 2878
       WAVEFRONT_PREFIX: pcf
-      WAVEFRONT_FOUNDATION: (( .properties.custom_tag_foundation.value ))
+      WAVEFRONT_FOUNDATION: (( ..cf.cloud_controller.system_domain.value ))
```

Comment:
This works, but you might want to leave it as a field the user can configure.  Seeing `foundation="prod"` (for example) in the Wavefront UI might be preferable to `foundation="sys.foo.company.com"`.

Reply:
Yeah, have a configurable tag for foundation is handy for creating multi-foundation views. You might want to be able to have an array of name/value tags. e.g. (foundation="prod" az="us-east1")